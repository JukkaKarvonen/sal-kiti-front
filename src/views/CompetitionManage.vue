<template>
  <div class="event">
    <Event v-on:update:createPermission="createPermission = $event" />
    <CompetitionForm />
  </div>
</template>

<script>
/**
 * Competition management view
 */
import CompetitionForm from "@/components/CompetitionForm.vue";
import Event from "@/components/EventInformation.vue";

export default {
  name: "CompetitionManageView",
  components: {
    CompetitionForm,
    Event
  },
  created() {
    document.title =
      this.$t("title.competition_manage") + " | " + this.$t("title.prefix");
  }
};
</script>
