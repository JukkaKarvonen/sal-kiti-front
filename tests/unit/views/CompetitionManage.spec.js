import { shallowMount, createLocalVue } from "@vue/test-utils";
import CompetitionManage from "@/views/CompetitionManage.vue";

import BootstrapVue from "bootstrap-vue";

const localVue = createLocalVue();
localVue.use(BootstrapVue);

describe("CompetitionManage.vue", () => {
  it("renders content", () => {
    const wrapper = shallowMount(CompetitionManage, {
      localVue
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
