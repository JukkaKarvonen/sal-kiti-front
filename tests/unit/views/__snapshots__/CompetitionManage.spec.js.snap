// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CompetitionManage.vue renders content 1`] = `
<div
  class="event"
>
  <event-stub />
   
  <competitionform-stub />
</div>
`;
